app/middleware/auth.js

```diff
@@ -4,7 +4,10 @@ const HttpStatus = require('http-status-codes')
 
 const auth = async (req, res, next) => {
   try {
-    const token = req.header('Authorization').replace('Bearer ', '')
+    const token = req.cookies.token || '';
+    if(!token) {
+      throw Error("unauthorized access")
+    }
     const decoded = jwt.verify(token, 'process.env.JWT_SECRET')
```

Comment:
'process.env.JWT_SECRET' should not be string

Reply:
It was already there. I'm confused about how earlier things have been working with this comparision!
I'll look around and update you regarding this.

Reply:
@Rupeshiya, Looks like we were this string as our secret while generating token too xD.
https://github.com/codeuino/social-platform-donut-backend/blob/b59ccca19cd0f44ffe10e59da456cd3546be61df/app/models/User.js#L264

I'll rectify and push both.

Reply:
Look at this: https://github.com/codeuino/social-platform-donut-backend/blob/development/app/controllers/user.js

Reply:
and others file as well

Reply:
Just checked that out! Looks like someone put these as a string at some places while as a variable at other places.